Documentation/ReleaseNotes.md

```diff
@@ -36,10 +37,189 @@ If importing the [Mixed Reality Toolkit NuGet packages](MRTKNuGetPackage.md), th
+
+A hand physics extension service has been added to allow for using physics interactions with the HoloLens 2 articulated hands ([#6573](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/6573)).
+
+![Hand physics](https://user-images.githubusercontent.com/1186832/68795768-77efdc00-0606-11ea-8fb9-b0e4191bdb05.gif)
+
+**Pinch Slider orientation**
+
+The Pinch Slider has been updated to orient TrackVisuals, TickMarks and ThumbRoot based on the sliderAxis orientation ([#6858](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/6858))
+
+![Y axis slier](https://user-images.githubusercontent.com/42405657/71687606-37a31380-2d96-11ea-84b5-ffe2368f8b57.JPG)
+![X axis slider](https://user-images.githubusercontent.com/42405657/71687640-4984b680-2d96-11ea-9f59-7732a91edd1b.JPG)
+
+**UX control prefabs updated to use PressableButton**
+
+The following prefabs are now using the PressableButton component instead of TouchHandler for near interaction ([7070](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/7070))
+
+- AnimationButton
+- Button
+- ButtonHoloLens1
+- ButtonHoloLens1Toggle
+- CheckBox
+- RadialSet
+- ToggleButton
+- ToggleSwitch
+- UnityUIButton
+- UnityUICheckboxButton
+- UnityUIRadialButton
+- UnityUIToggleButton
+
+![Updated UX prefabs](https://user-images.githubusercontent.com/36998103/72457296-30303100-37be-11ea-827f-d144a65348ff.gif)
+
 ### Known issues in 2.3.0
 
+**Issues with the Unity 2019.3 new XR platform on Windows Mixed Reality**
+
+The following issues are known when using the new XR platform and version **2.0.4 (preview.3)** of the Windows XR Plugin:
+
+- AirTap does not work on HoloLens (HoloLens 2 and 1st generation)
+- Pointers are using the wrong coordinate system on HoloLens 2 and immersive devices
+
+It is recommended to periodically check **Window** > **Package Manager** for newer versions of the Windows XR plugin.
+
+**Windows Mixed Reality gesture support on Unity 2019.3 when using the new XR platfom**
+
+This release of MRTK does not contain an implementation for Windows Mixed Reality gestures using the new XR platform. It will be added in a future version of MRTK.
+
+**Specifying the Depth Reprojection mode in the Windows Mixed Reality Camera Settings Provider is not supported on Unity 2019.3 and Windows XR plugin**
+
+This issue is expected to be fixed with upcoming releases of MRTK and the Windows XR plugin.
+
+**Mixed Reality Capture setting**
+
+This feature is currently not working correctly on Unity 2019.3.0f6. This issue impacts both the legacy and the new XR platform.
+
+**Long paths**
+
+When building on Windows, there is a MAX_PATH limit of 255 characters. Unity is affected by these limits and may fail to build a binary if its resolved output path is longer than 255 characters.
+
+This can manifest as CS0006 errors in Visual Studio that look like:
+
+> CS0006: Metadata file 'C:\path\to\longer\file\that\is\longer\than\255\characters\mrtk.long.binary.name.dll' could not be found.
```

Comment:
I think that this also can manifest with some other errors in MRTK non-NuGet code right? It might be helpful to have an example of that as well.

Reply:
True, though this is the most reported error.